mathics/builtin/datentime.py

```diff
@@ -665,14 +673,15 @@ def apply_makeboxes(self, datetime, gran, cal, tz, fmt, evaluation):
         "MakeBoxes[DateObject[datetime_List, gran_, cal_, tz_, fmt_], StandardForm|TraditionalForm|OutputForm]"
         # TODO:
         if fmt.sameQ(Symbol("Automatic")):
-            fmt = Expression("List", "DateTimeShort")
-        fmtds = Expression("DateString", datetime, fmt).evaluate(evaluation)
+            fmt = Expression(SymbolList, String("DateTimeShort"))
+        fmtds = Expression(SymbolDateString, datetime, fmt).evaluate(evaluation)
         if fmtds is None:
             return
         # tz = Expression("ToString", tz).evaluate(evaluation)
-        tz = int(tz.to_python())
-        tz = String(str(tz))
-        return Expression("RowBox", Expression("List", "[", fmtds, "  GTM", tz, "]"))
+        tz_string = String(str(int(tz.to_python())))
+        return to_expression(
```

Comment:
Why `to_expression` here?

Reply:
Because there are strings in the elements. I could wrap the string arguments in String, but here I was lazy. 

I will probably revise in the expression interface refactor branch where I can also list the element properties.